CMakeLists.txt

```diff
@@ -1,17 +1,45 @@
 cmake_minimum_required(VERSION 3.14)
-include(FetchContent)
 project(PyTorch)
+## libtorch requires C++-14
 set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+if(APPLE OR WIN32)
+  message(FATAL_ERROR "Configuring and building ITKPyTorch is not yet supported on macOS and Windows. See https://github.com/InsightSoftwareConsortium/ITKPyTorch/issues/4")
+endif()
 
+include(FetchContent)
+set(torch_fetch_URL "https://download.pytorch.org/libtorch/cu101/libtorch-cxx11-abi-shared-with-deps-1.5.1%2Bcu101.zip")
```

Comment:
In a future PR, can we build from source?

Reply:
That would be nice.  The instructions at https://github.com/pytorch/pytorch#from-source indicate that, ideally, we'd first have Anaconda, Python 3.6 or later, C++-14 compiler, CUDA 9.2 or higher, and cuDNN 7 or higher.  How many of those are already available to the Github CI?

Many of these dependencies are required even when using libtorch binaries, so likely the system-specific binaries that CMakeLists.txt knows about are not sufficient and we will have to bite this bullet sooner rather than later.

Reply:
I have added this as Issue https://github.com/InsightSoftwareConsortium/ITKPyTorch/issues/6.